src/options/options.js

```diff
@@ -86,16 +84,13 @@ document.getElementById('vocab-tabbed-header').addEventListener('click', (e) =>
     let targetContentDiv;
 
     // removing active class from any tab content div
-    Array.prototype.forEach.call(
-      document.getElementById('vocab-tabbed-contents').children,
-      (element) => {
-        element.classList.remove('active');
-        if (element.getAttribute('data-content-no') === tabNo) {
-          // saving the target content div
-          targetContentDiv = element;
-        }
-      },
-    );
+    Array.prototype.forEach.call(document.getElementById('vocab-tabbed-contents').children, element => {
```

Comment:
This project uses a column width of 120 charachters. It seems you are using 80 characters per line. We don't want to wrap code before 120 charachters.